js-webapp/README.md

```diff
@@ -9,40 +9,19 @@ _Wouldn't it be great if you could run your existing C/C++ code on the web with
 That way, loads more people would be able to see your results, interact with your algorithm, and apply it for their own
 purposes.
 
-In this blog, we'll show you how to take a simple algorithm written in C++ and make it available as a simple web
+In this blog, we'll show you how to take a simple algorithm written in C++ and make it available as a web
 application. Subsequent blogs in this series will expand on the current one by laying out more advanced topics,
 specifically how to [make the app interactive](anotherblog), how to [visualize the results](someotherblog), and
 how to [deal with long running tasks](yetanotherblog).
 
-So today's aim is to have a simple web app like the code snippet below. At first glance this is just some boilerplate to
-be able to use the ``newtonraphson.js`` library, but **the neat part is that the complete Newton-Raphson code is written
-in C++, not JavaScript**. With some trickery, we'll be able to use that C++ code from the browser, without the need to
-port it first!
+So today's aim is to have a simple web app that determines the _root_ of a mathematical function _x^3 - x^2 + 2_, i.e. the
+value of _x_ where _y = 0_. For this, we'll use the [_Newton-Raphson_ root finding method](https://www.youtube.com/watch?v=cOmAk82cr9M). Remember Newton? Quiet
```

Comment:
You now explain the equation, a picture like https://nlesc-jcer.github.io/cpp2wasm/images/equation.svg with a big arrow to -1,0 coordinate would help me here.

Reply:
good point. Maybe we should provide this feedback even before adding user interaction or long running tasks. Some simple d3 would do.